src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml/Given_FrameworkElement.cs

```diff
@@ -1246,19 +1246,6 @@ public async Task TestEventOrder()
 
 			Assert.AreEqual("Child Loaded", events[7]);
 			Assert.AreEqual("Parent Loaded", events[8]);
-
-
-#if HAS_UNO
-			if (events[9] == "Child LayoutUpdated")
-			{
-				Assert.AreEqual("Parent LayoutUpdated", events[10]);
-			}
-			else
-			{
-				Assert.AreEqual("Parent LayoutUpdated", events[9]);
-				Assert.AreEqual("Child LayoutUpdated", events[10]);
-			}
-#endif
```

Comment:
That's an unintended fix 😄 
But looks like we are doing less work now somehow (which also matches WinUI). So that's good :)

Reply:
That's great :-)